test/controllers/agent/RepresentationControllerSpec.scala

```diff
@@ -57,15 +61,13 @@ class RepresentationControllerSpec extends VoaPropertyLinkingSpec {
 
   behavior of "revokeClient method"
   it should "revoke an agent and display the revoke client page" in {
+    when(revokeClientPropertyPage.apply(any())(any(), any(), any())).thenReturn(Html(""))
```

Comment:
Was there a reason not to render the actual page and test against it?

Reply:
I don't think it we should be checking for html in unit tests (this is an old). The acceptance tests can do that.

Reply:
That wasn't Dan's advice.

Reply:
I guess there's not much logic on this page, so a single acceptance test would be enough. But if there is logic leading to different behaviour on the page, then unit tests are useful as we generally don't have acceptance tests for all possibilities, and unit tests catch errors sooner.

Reply:
We have used jsoup in recent pages to parse the page and then make various tests.

Reply:
This is what we've done is other frontend stories.

What was Dan's advice?

Reply:
One of Dan's handovers talked about developing a consistent pattern for testing pages - and we were using Jsoup for this. I'm saying that we have to do this way, but we should have a consistent approach so we are all on the same page (no pun intended).

Reply:
I think this is one that we might need to discuss as a dev team (so we can agree how we do this going forward). 

Seeing as there isn't any logic on this page, are you happy to give your thumb?

Reply:
why don't you remove "when(revokeClientPropertyPage.apply(any())(any(), any(), any())).thenReturn(Html(""))" 

this is not a good idea

Reply:
I think it is perfectly valid if we are not testing the html. We are just making sue that the mocked page loads, we are not interested in what is on the mocked page. Yes lets discuss for sure

Reply:
yes, but ... why? :)

Reply:
"if we are not testing the html" - you removed the test and I'm questioning it :)

Reply:
and returning empty html instead of a real page may hide errors - it's just about whether we test the content